From 0e3ad9a0f4c2375caec168036f2ddc3cebd2598a Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 21 Sep 2020 17:37:12 +0800
Subject: [PATCH] net: wireless: rockchip_wlan: rtl8821cs: Fix clang warning

drivers/net/wireless/rockchip_wlan/rtl8821cs/hal/phydm/phydm_dfs.c:1507:4:
warning: misleading indentation; statement is not part of the previous 'if'
[-Wmisleading-indentation]
                        dfs->pri_cond4 = 1;
                        ^

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I2aabcd339f71061872327712fbfeb39176073c69
---
 .../net/wireless/rockchip_wlan/rtl8821cs/hal/phydm/phydm_dfs.c  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8821cs/hal/phydm/phydm_dfs.c b/drivers/net/wireless/rockchip_wlan/rtl8821cs/hal/phydm/phydm_dfs.c
index f024844809fb..cc5703d5a2c0 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8821cs/hal/phydm/phydm_dfs.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8821cs/hal/phydm/phydm_dfs.c
@@ -1504,7 +1504,7 @@ void phydm_dfs_histogram_radar_distinguish(
 			dfs->pri_cond3 = 1;
 
 		/*@Cancel the condition that the abs between pri and pw*/
-			dfs->pri_cond4 = 1;
+		dfs->pri_cond4 = 1;
 
 		if (dfs->pri_hold_sum[5] <= dfs->pri_sum_g5_th)
 			dfs->pri_cond5 = 1;
-- 
2.35.3

